e2e/e2e_test.go

```diff
@@ -32,7 +32,7 @@ import (
 const (
 	wbLabelEqual     = "tier=whereabouts-scale-test"
 	testNamespace    = "kube-system"
-	rsCreateTimeout  = 30 * time.Second
+	rsCreateTimeout  = 600 * time.Second
```

Comment:
well, I don't understand how this fixed the timeout - apparently the traces were just very misleading. 

Never the less, I think 600 seconds is waaaay too much. Could you try it with 1 or 2 minutes please ? Seems more sensible.

Reply:
Let's look at the formula for how many replicas should be created.

```replicaCount = int32((envVars.numComputeNodes*maxPodsPerNode - (len(allPods.Items))) * envVars.fillPercentCapacity / 100)```

Let's say we have 80% fill capacity set for our e2e test. We have 5 compute nodes currently, and we know there's 110 pods per node maximum. That's about 550 pods, and 80% of that is 440 pods. Let's say the cluster has 40 pods prior to the e2e test, maybe doing whereabouts or multus backend work. This still totals 400 pods. 

I'm not sure 1 or 2 minutes is enough for this many replicas to come up. On my local environment, it takes several minutes to bring up just 100 pods. Let me know if I've made a mistake, happy to be proven wrong, but I think 10 minutes is reasonable for now.

Reply:
Pff. You might be right. Let's be safe then.